Реализовал верстку и логику

marsel1323 opened the pull request.

Commit:
Реализовал верстку и логику

Commit:
Remove sleep(3) from AppDelegate

andrew-kuzn-yndx commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,189 @@
+        
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet private var counterLabel: UILabel!
```
Лучше использовать модификатор weak для аутлетов. 
Вероятность retain cycle тут мала, но и нам ничего не стоит написать слабые ссылки)

andrew-kuzn-yndx commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,189 @@
+        
+        let action = UIAlertAction(
+            title: result.buttonText,
+            style: .default) { _ in
```
Когда мы обращаемся к self внутри замыкания, то лучше использовать [weak self], чтобы избежать неожиданного поведения
Вот тут подробно: https://codeswift.ru/weak-self-closure-swift/

andrew-kuzn-yndx commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,189 @@
+
+        imageView.layer.masksToBounds = true
+        imageView.layer.borderWidth = 8
+        imageView.layer.borderColor = isCorrect ? UIColor.ypGreen.cgColor : UIColor.ypRed.cgColor
```
Красивый синтаксис, молодец!

Commit:
Поправил код согласно комментариям с гитхаба
Поправил верстку: сделал видимым счетчик вопросов, добавил скругления для постера, убрал нижний констрейт чтобы текст вопроса отображался полностью
